zen-quotes-app/src/app/authors/authors.component.html

```diff
@@ -0,0 +1,27 @@
+<section>
+  <ul class="flex flex-wrap justify-between mb-4">
+    <li *ngFor="let author of authors | slice: 1:authorsQuantity">
```

Comment:
It's a good practice to always use a trackBy function (see https://angular.io/api/common/NgForOf#description). 

Challenge: would you be able to add one? 

Thanks! 🙏 🙇

Reply:
Thank you very much! Done 🙏